src/rail/estimation/algos/deepdisc.py

```diff
@@ -1,4 +1,31 @@
+import numpy as np
 from rail.estimation.estimator import CatEstimator, CatInformer
+from rail.core.data import TableHandle
+from rail.core.common_params import SHARED_PARAMS
+from detectron2.config import LazyConfig, get_cfg
+import detectron2.solver as solver
+import detectron2.data as data
```

Comment:
I might suggest that we do something like `import detectron2.data as d2data` or something. The term `data` seems like it might be easy to overload. It's also possible that other methods would be shadowed by this. i.e. if a class `foo` has a method called `data`, then using this name for the import, it wouldn't be clear what the interpreter should do if you call `foo.data`.

Reply:
Good point.  I can change that